### PR TITLE
chore: Move all scattered SVGs to shared location

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
More SVGs that are scattered around our code base into the common `/src/internal/svg` folder
**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: src/internal/svg/badgeSvg.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { fill } from '../../styles/theme';
+
+export const badgeSvg = (
+  <svg
+    role="img"
+    aria-label="ock-attestation-icon"
```
Comment (Zizzamia) [inline src/internal/svg/badgeSvg.tsx:6]: ock-badgeSvg instead

File: src/internal/svg/caretDownSvg.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { fill } from '../../styles/theme';
+
+export const caretDownSvg = (
+  <svg
+    data-testid="ockTokenSelectButton_CaretDown"
```
Comment (Zizzamia) [inline src/internal/svg/caretDownSvg.tsx:5]: same here and in other places.

Comment (cpcramer) [reply src/internal/svg/caretDownSvg.tsx:5]: okay on it

Comment (cpcramer) [reply src/internal/svg/caretDownSvg.tsx]: Updating the existing data-testid fields and the aria-label fields.

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**